EasyModbusTCPCore/ModbusClient.cs

```diff
@@ -73,13 +74,13 @@ public ModbusClient()
 		/// <summary>
 		/// Establish connection to Master device in case of Modbus TCP. Opens COM-Port in case of Modbus RTU
 		/// </summary>
-		public void Connect()
+		public async Task ConnectAsync()
```

Comment:
I would suggest to add CancellationToken support to Async Methods and also to evaluat it as well and do not trust the third party is doing it.